pom.xml

```diff
@@ -55,9 +55,9 @@
   </distributionManagement>
 
   <properties>
-    <maven.version>3.1.1</maven.version>
     <javaVersion>7</javaVersion>
-    <aether.version>0.9.0.M2</aether.version>
+    <maven.version>3.1.1</maven.version>
+    <resolver.version>1.6.2</resolver.version>
```

Comment:
I'm not sure about minimal resolver version here

Reply:
resolver should be binary compatible, at least the used bits, but we can lower if really needed (hopefully not)